packages/firebase_dynamic_links/lib/firebase_dynamic_links.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library firebase_dynamic_links;
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+
+part 'src/android_parameters.dart';
+part 'src/dynamic_link_parameters.dart';
+part 'src/dynamic_link_parameters_options.dart';
+part 'src/firebase_dynamic_links.dart';
+part 'src/google_analytics_parameters.dart';
+part 'src/ios_parameters.dart';
+part 'src/itunes_connect_analytics_parameters.dart';
+part 'src/navigation_info_parameters.dart';
+part 'src/short_dynamic_link.dart';
+part 'src/social_meta_tag_parameters.dart';
```

Comment:
I have not seen this "one part = one file = one class" style before with Dart. Seems like a Java thing, really. But I have no strong opinion here.